### PR TITLE
[MRG]🎨 refractor modelci hub converter structure

### LINKED ISSUE #232: [converter] Refractor file structure of modelci hub converter
#### Software and Hardware Versions 
ModelCI v1.x.x, CUDA Version vx.x.x, GPU device used...

#### Other Information 
Things you tried, stack traces, related issues, suggestions on how to fix it...

### PR DESCRIPTION
**Pull request by univerone**
Closes #232

#### What has been done to verify that this works as intended?

- Split converter into several modules, provides a common `convert` API, here is an example usage
```python
torch_model = torch.hub.load('pytorch/vision:v0.6.0', 'resnet50', pretrained=True)
inputs = [IOShape([-1, 3, 224, 224], dtype=float, name='INPUT__0', format=ModelInputFormat.FORMAT_NCHW)]
outputs = [IOShape([-1, 1000], dtype=float, name='probs')]
convert(torch_model, 'pytorch', 'onnx', save_path=onnx_path, inputs=inputs, outputs=outputs)
```
- Refractor current converter test to coverage more 

#### Before submitting this PR, please make sure you have:

- [x] run `python -m pytest tests/` and confirmed all checks still pass.
- [x] verified that any code or assets from external sources are properly credited.

### CODE REVIEW

File: modelci/hub/converter/onnx/__init__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python3
```
Comment (HuaizhengZhang) [inline modelci/hub/converter/onnx/__init__.py:1]: two deep dir

something hub/converter/onnx_converter.py is enough

**Review by HuaizhengZhang — COMMENTED** [review]
The unit test is very good.

**Review by huangyz0918 — APPROVED** [review]
Great job! @univerone Thanks for the great contributions!

**Review by YuanmingLeee — APPROVED** [review]
lgtm. Things to be noted that we do not want too deep packages. Try put:
```
hub/convert/onnx/convert.py -> hub/convert/to_onnx.py
hub/convert/pytorch/convert.py -> hub/convert/to_pytorch.py
...
```

**Review by HuaizhengZhang — APPROVED** [review]

**Pull request merged**